doc/cabal-commands.rst

```diff
@@ -184,6 +192,17 @@ cannot be excluded for technical reasons).
 
     $ cabal v2-repl --build-depends vector --no-transitive-deps
 
+Scripts can be passing the path to the script as the target.
+
+::
+
+    $ cabal v2-repl script.hs
+
+If the name of the script matches another target, the other target is preferred.
```

Comment:
In what context may that arise? I suspect in a project (e.g. cabal.project exists) and `script.hs` is part of the cabal project?

Reply:
Also, the same caveat will apply for `v2-build` as well, right?

Reply:
I should probably remove the .hs because script doesn't necessarily have to end with .hs. so you could have a target called `mangler` and a script called `mangler` or whatever. You're right that it can occur for the other commands as well.